### ISSUE TITLE
Clarify plans/vision for Peril

### ISSUE THREAD

**Issue by jtreanor** [danger/peril#408]
Hi there!

I've recently discovered Peril having used Danger for some time and its a really nice project!

Reading [`VISION.md`](https://github.com/danger/peril/blob/master/VISION.md) helped me a lot to understand the background of the project but it left me wondering how the project will progress in future as it seems to be reasonably feature complete in that document.

Is it still planned for Peril to be run as a hosted service (similar to how [Hound](https://houndci.com)) works?

I know this is a volunteer run open source project and I am certainly not looking for an ETA but I would love to hear what the current plans are and if there is anything contributors can do to help accelerate the goal.

**Comment by orta** [comment]
Yeah, that was my original plan - but once GitHub Actions came out I'm not really sure if it's worth my time. 

So, I've left that idea kinda on the backburner realistically.

**Comment by jtreanor** [comment]
Thanks for the quick response @orta!

I hadn't consider using Github Actions for Danger. That could be a great use case. Is there an established way of using Github Actions with Danger?

I guess it's a good time for some patience to see how Github Actions matures and makes sense for production.

**Comment by orta** [comment]
Yeah, I got both Danger JS and Danger Swift working with GitHub Actions before the beta, but due to the current beta limitations (open source repos can only have push web hooks) then I've never been able to use it in production anywhere (I only have OSS repos, and push web hooks are a tiny part of the story for me) 

So it works, but I can't guarantee it's 100% 

--- 

Worth noting: Artsy and myself (CocoaPods/Danger) are heavily invested in Peril. The lack of work on the server itself has been because it's working, and now work happens in our user-land instead https://github.com/artsy/peril-settings/graphs/contributors

**Comment by jtreanor** [comment]
Ah, interesting. I'm OSS only too (I work on WordPress mobile) so that limitation would probably apply to me also.

Thats good to hear that Peril is still actively used. I think it may make sense for us to adopt Peril and move to Github Actions if it becomes more viable in future.
